Marlin/ultralcd_impl_DOGM.h

```diff
@@ -598,16 +587,16 @@ static void lcd_implementation_status_screen() {
       }
     #endif
 
-    if (lcd_strlen_P(pstr) > LCD_WIDTH - 2 - vallen) rows = 2;
+    if (uxg_GetUtf8StrPixelWidthP (pu8g->getU8g(), pstr) > DOG_CHAR_WIDTH * (LCD_WIDTH - 2 - vallen)) rows = 2;
 
     const float kHalfChar = (DOG_CHAR_HEIGHT_EDIT) / 2;
     float rowHeight = u8g.getHeight() / (rows + 1); // 1/(rows+1) = 1/2 or 1/3
 
-    u8g.setPrintPos(0, rowHeight + kHalfChar);
+    lcd_moveto (0, rowHeight + kHalfChar);
     lcd_printPGM(pstr);
     if (value != NULL) {
       lcd_print(':');
-      u8g.setPrintPos((lcd_width - 1 - vallen) * char_width, rows * rowHeight + kHalfChar);
+      lcd_moveto ((lcd_width - 1 - vallen) * char_width, rows * rowHeight + kHalfChar);
```

Comment:
As a matter of code style, please remove spaces between the function name and the opening parenthesis.